### PR TITLE
Add administration service for deleting service acquisition

### PR DESCRIPTION
**Pull request by vramperez**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -198,7 +199,7 @@ To display brief information: `./cli -h` or `./cli --help`. In order to get info
 
 Proxy's api is in port **9000** by default and root path **/accounting_proxy/..**.
 
-### POST ../buys
+### POST .../newBuy
 
 Use by the store to notify a new buy:
```
Comment (fdelavega) [inline README.md]: Used by ...

File: README.md [hunk 2]
```diff
@@ -220,7 +221,21 @@ Use by the store to notify a new buy:
 * `unit`: accounting unit (`megabyte`, `call`, ...).
 * `recordType`: type of accounting.
 
-### POST ../urls
+### POST .../deleteBuy
+
+Use by the store to notify a deleted buy:
```
Comment (fdelavega) [inline README.md]: Used by ...

File: README.md [hunk 3]
```diff
@@ -220,7 +221,21 @@ Use by the store to notify a new buy:
...
+ "customer": "..."
+}
+```
+* `orderId`: order identifier.
+* `productId`: product identifier.
+* `customer`: the userId os the customer.
+
+### POST .../urls
 
 Use by the store to check if an URL is valid. This requests require the "authorization" header with a valid access token from the IdM and the user must be an administrator of the service.
```
Comment (fdelavega) [inline README.md]: Used by ...

**Comment by vramperez** [comment]
New commit

**Pull request merged**